.github/workflows/chapter-4-contracts-workflow.yml

```diff
@@ -29,7 +29,7 @@ jobs:
         with:
           dotnet-version: 8.0.x
       - name: Add Evolutionary Architecture Nuget Source
-        uses: evolutionary-architecture/evolutionary-architecture-by-example/.github@main
+        uses: evolutionary-architecture/evolutionary-architecture-by-example/.github@feature/add_signature_value_object
```

Comment:
Please change it back to main (I assume this is the leftover from testing)

Reply:
I'm going to revert is after merge otherwise pipeline will fail.